recipe/meta.yaml

```diff
@@ -55,3 +88,5 @@ about:
 extra:
   recipe-maintainers:
     - jdb78
+  skip-lints:
+    - missing_wheel
```

Comment:
does adding wheel to the dependencies cause any problem?

Reply:
No, but @JeanChristopheMorinPerso has already pointed https://github.com/AnacondaRecipes/prince-feedstock/pull/1#discussion_r1244001638 that it's redundant for `poetry-core`

Reply:
cool, I'll send some PRs to the linter when I find some time, to fix the false positives for `pip check` when it is explicitly disabled, this wheel redundancy and maybe the blas dependency declarations